conf/messages.cy

```diff
@@ -66,8 +66,10 @@ technical.heading=Gwybodaeth dechnegol am hygyrchedd y gwasanaeth hwn
 technical.commitment=Mae CThEM wedi ymrwymo i wneud y gwasanaeth hwn yn hygyrch, yn unol â Rheoliadau Hygyrchedd Cyrff Sector Cyhoeddus (Gwefannau a Chymwysiadau Symudol) (Rhif 2) 2018.
 technical.compliance.full='Mae’r gwasanaeth hwn yn cydymffurfio’n llawn â '
 technical.compliance.partial=This service is partially compliant with the
+technical.compliance.noncompliant=ABCDEFG
```

Comment:
Is this just a placeholder?

Reply:
Replaced placeholder with English. Welsh translations to be added as part of separate ticket.